src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,12 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        boolean res = ((a && b && !c && !d)
```

Comment:
Good start, but try to think how you can make solution less complicated.
Hint:
- you don't need separate variable "res"
- you can use fewer boolean operations

Reply:
you can read something about XOR operation

Reply:
done